### DISCUSSION TITLE
-

### DISCUSSION THREAD

**Discussion by frutas-fruit** [tippesi/Atlas-Engine#12] [General]
.

**Comment by tippesi** [comment]
Hi,

thanks for the feedback!

I think this might be related to the older SDL version that is being used here. I will try to update it to the newest version in the upcoming days. If you don't want to wait until then, you can try to use the newest SDL version by yourself and just paste it to the dependencies folder. If there are no breaking changes on the SDL side, you should be fine.

I will get back to you when I have integrated SDL 2.0.20 into the engine

**Reply by tippesi** [reply]
SDL 2.0.20 is now integrated and there are a few other fixes in regards to Linux. I hope it works. You might need to change the asset directory path in the demo application when you hopefully get it to build, in order to load the shaders.

**Reply by tippesi** [reply]
Sorry for the late reply. This seems to be an issue on the [Assimp](https://github.com/assimp/assimp/issues/4334) side. The newest commit now makes the Assimp integration optional. To turn it off, you need to disable the ATLAS_ASSIMP CMake option. Furthermore, if you want to load model data using tinyobj, you just need to supply mesh data to a mesh. You can find an example of how this works in the [Assimp Modelloader](https://github.com/tippesi/Atlas-Engine/blob/master/src/loader/ModelLoader.cpp). The setup of a mesh is then similar to what you find in the demo application
